azure-functions-codegen/src/func/mod.rs

```diff
@@ -427,19 +418,17 @@ pub fn attr_impl(args: TokenStream, input: TokenStream) -> TokenStream {
         let (_, value) = args.list.iter().find(|(k, _)| k == "name").unwrap();
         match value {
             Lit::Str(s) => {
-                value
-                    .span()
-                    .unstable()
-                    .error(match s.value().as_ref() {
-                        RETURN_BINDING_NAME => {
-                            "cannot bind to a function without a return value".to_string()
-                        }
-                        v => format!(
-                            "cannot bind to '{}' because it is not a binding parameter of the function",
-                            v
-                        ),
-                    })
-                    .emit();
+                let foo = match s.value().as_ref() {
```

Comment:
`foo` is a placeholder name that generates a warning for clippy.

Perhaps `message` instead?

Reply:
Whoops. How embarrassing!